packages/identity/package.json

```diff
@@ -46,8 +47,8 @@
     "fetch-mock": "9.11.0",
     "ganache": "npm:@celo/ganache@7.8.0-unofficial.0",
     "jest": "^29.7.0",
-    "old-celo-base": "npm:@celo/base@5.0.0",
-    "old-identity-sdk": "npm:@celo/identity@1.5.2"
+    "old-celo-base": "npm:@celo/base@1.5.2",
+    "old-identity-sdk": "npm:@celo/identity@5.1.0"
```

Comment:
appearently this name "old-identity-sdk" causes socket.dev to think this is referencing an npm package called old-identity-sdk which  has compromises in it. so i suggest renaming this to  `old-celo-identity-sdk`

Reply:
(for context)

On [socket.dev](http://socket.dev/), we're seeing a warning that the `@celo/identity` NPM package makes use of a malicious NPM dependency called `old-identity-sdk`.

It's not really a vulnerability and probably a misunderstanding (on [socket.dev](http://socket.dev/)'s part), because `old-identity-sdk` is just an alias. 

But, while we're at it, we may as well change the alias name to mitigate this false alert.

<img width="250" src="https://github.com/celo-org/social-connect/assets/46296830/db8de441-54c5-43b0-b32b-004df56d4402">

Reply:
Fixed here https://github.com/celo-org/social-connect/pull/207/commits/2babceff620085d5d2d7155c4ff7b69987fb4062